tox.ini

```
@@ -87,7 +87,7 @@ description = Run integration tests
 description = Run scenario tests
 deps =
     pytest
-    ops-scenario>=6
+    ops-scenario
```

While we're at it, if not too much extra effort, can we change this to
```suggestion
    ops[testing]
```
?